PostBy/View Controllers/ComposeViewController.m

```diff
@@ -41,15 +41,46 @@ - (void)viewDidLoad {
 }
 
 - (void)setUpUI {
-    self.postTextField.text = @"";
+    if (self.postVMToUpdate != nil) {
+        self.postTextField.text = self.postVMToUpdate.postText;
+        if (self.postVMToUpdate.hideLocation)
+            [self hideLocationTapped:nil];
+        if (self.postVMToUpdate.hideUsername)
+            [self hideUsernameTapped:nil];
+        if (self.postVMToUpdate.hideProfilePic)
+            [self hideProfilePicTapped:nil];
+    } else {
+        self.postTextField.text = @"";
+    }
     
     // Setting up the border of our UITextView
     self.postTextField.layer.borderWidth = 0.5;
```

Comment:
If the number is only used once in the file, it is fine.
If the number will be used multiple times in the file, it is better to create a variable for it.
If the number will be referenced across multiple files, it is much better to create a global constant for it.